src/app/frontend/application/detail/info.html

```diff
@@ -0,0 +1,77 @@
+<!--
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<kd-info-card>
+  <kd-info-card-header>[[Details|Header in a detail view]]</kd-info-card-header>
+  <kd-info-card-section>
+    <kd-object-meta-info-card object-meta="::$ctrl.application.objectMeta">
+    </kd-object-meta-info-card>
+    <kd-info-card-entry title="[[Selector|Label 'Selector' for the application's labels list on the application details page.]]">
+      <kd-labels labels="::$ctrl.application.selector"></kd-labels>
+    </kd-info-card-entry>
+    <kd-info-card-entry title="[[Type|Type 'Type' for the application's Type on the application details page.]]">
+      {{ ::$ctrl.application.type }}
+    </kd-info-card-entry>
+    <kd-info-card-entry title="[[Assembly Phase|Assembly Phase 'Assembly Phase' for the application's Assembly Phase on the application details page.]]">
```

Comment:
The assemblyPhase will be useful for computing the application's status, but it makes little sense to display it as a field.